tpcx_bb/queries/q28/tpcx_bb_query_28_sql.py

```diff
@@ -36,34 +36,33 @@
 N_FEATURES = 2 ** 23  # Spark is doing 2^20
 ngram_range = (1, 2)
 lowercase = True
-preprocessor = None
+preprocessor = lambda s:s
```

Comment:
Why do we need a no-op lambda? Do we need to file a cuml issue?

Reply:
Switched from no-op to op with commit https://github.com/rapidsai/tpcx-bb/pull/104/commits/02057c55cf89de3ecbd72c1c031c3f92257283a2 . 


Currently, a user cant switch off different pre-processing in cuML vectorizers without a `no-op` like function as `preprocessor `  because with None (default) it is coupled to get the same behavior as the default regex tokenizer in sKLearn. 

We don't support the `token_pattern ` parameter as  regex-based tokenization will have performance issues on GPU's.  



https://github.com/scikit-learn/scikit-learn/blob/0fb307bf39bbdacd6ed713c00724f8f871d60370/sklearn/feature_extraction/text.py#L334-L345

Happy to discuss this in more detail but it should be not a problem in this PR anymore.

Reply:
Ah, understood. Thanks for the clarification